packages/fhir-care-team/src/components/CreateEditCareTeam/Form.tsx

```diff
@@ -54,7 +54,7 @@ const CareTeamForm: React.FC<CareTeamFormProps> = (props: CareTeamFormProps) =>
   return (
     <Row className="content-section user-group">
       {/** If email is provided render edit group otherwise add group */}
-      <PageHeader
+      <SimplePageHeader
```

Comment:
@ciremusyoka should we also just use the rich page header here so that it matches the design.
![image](https://github.com/onaio/fhir-web/assets/28119869/17653284-0346-4ec6-88cd-6114a3d6f081)